resource-managers/kubernetes/README.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: global
+title: Spark on Kubernetes Development
```

Comment:
at the bottom of the page we should link to the user docs for how to use it as a natural continuation from the dev setup docs